extension/src/popup/components/accountCreator/PasswordForm/index.tsx

```diff
@@ -0,0 +1,166 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import {
+  Input,
+  Checkbox,
+  Link,
+  Button,
+  Card,
+  Heading,
+  Icon,
+} from "@stellar/design-system";
+import { Field, FieldProps, Form, FormikErrors, FormikTouched } from "formik";
+import { useTranslation } from "react-i18next";
+
+import {
+  Onboarding,
+  OnboardingButtons,
+  OnboardingOneCol,
+} from "popup/components/Onboarding";
+import { authErrorSelector } from "popup/ducks/accountServices";
+
+import "./styles.scss";
+
+export interface FormValues {
+  password: string;
+  confirmPassword: string;
+  termsOfUse: boolean;
+}
+
+interface PasswordFormProps {
+  isValid: boolean;
+  dirty: boolean;
+  isSubmitting: boolean;
+  errors: FormikErrors<FormValues>;
+  touched: FormikTouched<FormValues>;
+  values: FormValues;
+  handleSubmit?: (values: FormValues) => void;
+}
+
+export const initialValues: FormValues = {
+  password: "",
+  confirmPassword: "",
+  termsOfUse: false,
+};
+
+export const PasswordForm = ({
```

Comment:
this is just breaking out some code I wrote for the Create Wallet form into a reusable component